packages/bbui/src/Form/Core/Select.svelte

```diff
@@ -8,15 +8,18 @@
   export let disabled = false
   export let error = null
   export let options = []
+  export let callbackOptionValue = null
   export let getOptionLabel = option => option
   export let getOptionValue = option => option
+  export let getOptionIcon = option => option
```

Comment:
We'll need to change the default here to be no icon, because the standard-components form components also inherit the core variants, and if the default `getOptionIcon` returns a value of `option` then I have a feeling that this will render nonsensical string icons for every option in selects in client apps!

So I think just default to no icons in the core `Select`.
```js
export let getOptionIcon = option => null
```